nginx.conf

```diff
@@ -441,3 +441,15 @@ server {
   # Redirect rule so that /python/docs actually leads somewhere
   rewrite ^/python/docs/?$ /python/docs/quick-start;
 }
+
+server {
+    listen 80;
+
+    server_name ~^(?<subdomain>[^.]+)\.preview\.docs\.apify\.com$;
+
+    location / {
+        rewrite ^/(.*)/$ https://s3.amazonaws.com/apify-docs-preview/$subdomain/$1/index.html permanent;
```

Comment:
As discussed in person - let's try to use `proxy_pass` here so that it doesn't require enabling public read on the bucket.